### PR TITLE
Improve flow of 1st push to Bitbucket & use `hg config --local` command.

### PR DESCRIPTION
**Pull request by douglatornell**

### CODE REVIEW

File: 02-collab.md [hunk 1]
```diff
@@ -19,42 +19,93 @@ Systems like Mercurial allow us to move work between any two repositories.
...
+From within your `planets` directory,
+issue a command like:
 
-    hg push ssh://hg@bitbucket.org/vlad/planets
+~~~ {.bash}
+$ hg push https://bitbucket.org/vlad/planets
+~~~
+
+It's similar to the one shown under "I have an existing project" on the Bitbcucket page,
+but the URL should have your Bitbucket user name in it,
```
Comment (kynan) [inline 02-collab.md]: but the command above does not

Comment (kynan) [inline 02-collab.md]: Sorry, you mean instead of `vlad`?

**Comment by kynan** [comment]
Looks good to me.

File: 02-collab.md (hunk 1 above)
Comment (douglatornell) [inline 02-collab.md]: Yes. Thanks for catching that. I'll fix it in a while.

**Pull request merged**